### ISSUE TITLE
RSA certificates not being created during the wazuh-manager package installation

### ISSUE THREAD

**Issue by teddytpc1** [wazuh/wazuh#24441]
## Description
We found that the VD was not working properly during the [Release 4.8.1 - RC 1 - Demo use cases] (https://github.com/wazuh/wazuh/issues/24440) test.
The `indexer-connector` does not find the username and password in the Keystore and is throwing the following messages:

```
2024/07/05 14:03:18 indexer-connector: WARNING: No username and password found in the keystore, using default values.
2024/07/05 14:03:18 wazuh-modulesd:syscollector: INFO: Evaluation finished.
2024/07/05 14:03:18 indexer-connector: WARNING: IndexerConnector initialization failed for index 'wazuh-states-vulnerabilities-wazuh1', retrying until the connection is successful.
2024/07/05 14:03:19 wazuh-modulesd:vulnerability-scanner: INFO: Vulnerability scanner module started
2024/07/05 14:03:19 indexer-connector: WARNING: Failed to sync agent '001' with the indexer.
2024/07/05 14:03:31 wazuh-syscheckd: INFO: (6009): File integrity monitoring scan ended.
2024/07/05 14:03:31 wazuh-syscheckd: INFO: FIM sync module started.
2024/07/05 14:03:31 wazuh-syscheckd: INFO: (6012): Real-time file integrity monitoring started.
2024/07/05 14:03:34 wazuh-modulesd:azure-logs: INFO: Finished Log Analytics collection for request 'request_1808490687'.
2024/07/05 14:03:34 wazuh-modulesd:azure-logs: INFO: Finished Log Analytics collection for the domain 'wazuh.com'.
2024/07/05 14:03:36 wazuh-modulesd:vulnerability-scanner: ERROR: VulnerabilityScannerFacade::initEventDispatcher: [json.exception.parse_error.101] parse error at line 1, column 217: syntax error while parsing value - invalid lit
eral; last read: '"35c19963a6f534ecd85b866cdb168a29086ea2a8"}}<U+0002>'; expected end of input
2024/07/05 14:03:38 wazuh-modulesd:vulnerability-scanner: ERROR: VulnerabilityScannerFacade::initEventDispatcher: [json.exception.parse_error.101] parse error at line 1, column 225: syntax error while parsing value - invalid lit
eral; last read: '"0dee6952600be0d03e8172e0a89ac843c36e2b1b"}}<U+0003>'; expected end of input
2024/07/05 14:03:39 wazuh-modulesd:vulnerability-scanner: ERROR: VulnerabilityScannerFacade::initEventDispatcher: [json.exception.parse_error.101] parse error at line 1, column 233: syntax error while parsing value - invalid literal; last read: '"d6a9a0661515df5f05c694ed018ba6870f75cb8f"}}<U+0003>'; expected end of input
```

I have manually executed the `wazuh-keystore` commands and the changes are not being saved in the database:

```
[root@wazuh-manager-master-0 keystore]# /var/ossec/bin/wazuh-keystore -f indexer -k username -v admin
[root@wazuh-manager-master-0 keystore]# echo $?
0
[root@wazuh-manager-master-0 keystore]# /var/ossec/bin/wazuh-keystore -f indexer -k password -v 4aYbz4ab...
[root@wazuh-manager-master-0 keystore]# echo $?
0
```
```
[root@wazuh-manager-master-0 keystore]# ls -l
total 36
-rw-r--r-- 1 root root     0 Jul  5 17:44 000034.log
-rw-r--r-- 1 root root    16 Jul  5 17:44 CURRENT
-rw-r--r-- 1 root root    36 Jul  5 13:37 IDENTITY
-rw-r--r-- 1 root root     0 Jul  5 13:37 LOCK
-rw-r--r-- 1 root root     0 Jul  5 17:44 LOG
-rw-r--r-- 1 root root   185 Jul  5 17:44 MANIFEST-000035
-rw-r--r-- 1 root root 11458 Jul  5 17:44 OPTIONS-000033
-rw-r--r-- 1 root root 11458 Jul  5 17:44 OPTIONS-000037
```

The `wazuh-manager` package installation is not creating the certificates because the following command is returning 0, although the certificates (`/var/ossec/etc/keystore.key` and `/var/ossec/etc/keystore.cert`) do not exist:
https://github.com/wazuh/wazuh/blob/889f0c61f68d83ef6f5fc6936dcfc22bae11fa24/src/init/inst-functions.sh#L1320

**Comment by sebasfalcone** [comment]
@Dwordcito I've assigned you the issue based on the conversation we had about the removal of certificates on the keystore module

**Comment by Dwordcito** [comment]
This will be fixed in 4.8.1, https://github.com/wazuh/wazuh/issues/24441 Certificates will no longer be necessary.
